src/licensedcode/data/rules/fftpack-2004_2.RULE

```diff
@@ -0,0 +1,45 @@
+---
+license_expression: fftpack-2004
+is_license_text: yes
+ignorable_copyrights:
+    - Copyright (c) 2004 the University Corporation for Atmospheric Research 'UCAR
+ignorable_holders:
+    - the University Corporation for Atmospheric Research 'UCAR
+ignorable_authors:
+    - NCAR's Computational and Information Systems Laboratory, UCAR, www.cisl.ucar.edu
+ignorable_urls:
+    - http://www.cisl.ucar.edu/
+---
+
+Copyright (c) 2004 the University Corporation for Atmospheric
+Research ("UCAR"). All rights reserved. Developed by NCAR's
+Computational and Information Systems Laboratory, UCAR,
+www.cisl.ucar.edu.
```

Comment:
Why keep this copyright and header? IMHO this does not bring anything ... We should always try to avoid copyrights in rules and licenses, unless this is the copyright OF the license itself and it is PART of the license

Reply:
missed this, removing.